content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This can be enabled through the `extendedIndexes` Preview feature in versions `3.5.0` and later, and is in General Availability in versions `4.0.0` and later.
 
-Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` preview feature.
+Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` Preview feature.
 
 <Admonition type="warning">
 
-These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features, so they need to be explicitly enabled via preview features.
+These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features.
```

Comment:
Can we add one more sentence explaining why it would be a breaking change if you already have that feature enabled? I guess because new version implements the feature in a diff way?

Reply:
Agree - I'll improve this as I write the corresponding section in the upgrade guide

Reply:
Added a link to the upgrade section of the page which gives more detail